### ISSUE TITLE
Spurious temperatures at mesh interface with STRATIFICATION=T

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#5662]
The attached case was derived from the Natural_Convection series.  The bottom has been opened to eliminate the pressure zone effect.  There are two meshes, one on top of the other.  With STRATIFICATION=T (default) there is a spurious temperature error at the mesh interface.  With STRATIFICATION=F the problem goes away.  Yet I can't find a simple problem with RHO_0(K) anywhere in the code.

[strat.fds.txt](https://github.com/firemodels/fds/files/1435653/strat.fds.txt)

**Comment by rmcdermo** [comment]
I've checked the Z momentum equation at the mesh interface with these lines

```
FVZ(I,J,K) = 0.25_EB*(VOMX - UOMY) - GZ(I) + RRHO*(GZ(I)*0.5_EB*(RHO_0(K)+RHO_0(K+1)) - VTRM)

if (nm==1 .and. i==8 .and. j==1 .and. k==64) print *,1, FVZ(i,j,k)
if (nm==2 .and. i==8 .and. j==1 .and. k==0 ) print *,2, FVZ(i,j,k)
```

and FVZ seem to be identical across the mesh interface.  This is consistent with my printouts of RHO_0, etc., which all seem fine.

**Comment by rmcdermo** [comment]
This should be fixed with #5697.

**Comment by sbenkorichi** [comment]
Well done ! :+1:
Confirmed from my end !
